deploy/config/epp-prefix-cache-tracking-config.yaml

```diff
@@ -3,20 +3,29 @@
 apiVersion: inference.networking.x-k8s.io/v1alpha1
 kind: EndpointPickerConfig
 plugins:
-- type: single-profile-handler
-- type: decode-filter
-- type: prefix-cache-scorer
-  parameters:
-    mode: cache_tracking
-    kvCacheRedisAddr: ${REDIS_HOST}:${REDIS_PORT}
-- type: load-aware-scorer
-- type: max-score-picker
+  - type: single-profile-handler
+  - type: decode-filter
+  - type: prefix-cache-scorer
+    parameters:
+      mode: cache_tracking
+      indexerConfig:
+        tokenProcessorConfig:
+          blockSize: 64                 # must match vLLM block size
+          hashSeed: "42"                # must match vLLM PYTHONHASHSEED env var
+        kvBlockIndexConfig:
+          enableMetrics: true                   # enable kv-block index metrics (prometheus)
+          metricsLoggingInterval: 60000000000   # log kv-block metrics as well (1m in nanoseconds)
```

Comment:
it is possible to set it a human readable format?
seconds for example? maybe use duration format to allow any unit? (e.g., 60s, 1m, etc..)
you don't expect a reader to start calculations of nanoseconds..

Reply:
That was the original intention, but generally this field is useless, useful for a quick example. The metrics exposed are accumulative and make sense when sequenced over-time. The useful metrics analysis should be done through prometheus.

I can emit this from the default configuration, or change it if you insist.